pkg/diff/plan.go

```diff
@@ -17,6 +17,7 @@ const (
 	MigrationHazardTypeIndexDropped                MigrationHazardType = "INDEX_DROPPED"
 	MigrationHazardTypeImpactsDatabasePerformance  MigrationHazardType = "IMPACTS_DATABASE_PERFORMANCE"
 	MigrationHazardTypeIsUserGenerated             MigrationHazardType = "IS_USER_GENERATED"
+	MigrationHazardTypeVersionUpgrade              MigrationHazardType = "UPGRADING_ENTITY_VERSION"
```

Comment:
I think it's okay to make this extension specific, like `UPGRADING_EXTENSION_VERSION`. Seems rare enough and important enough to make specific